brion/plugin/morphologyZeroEQ.h

```diff
@@ -0,0 +1,93 @@
+/* Copyright (c) 2017, EPFL/Blue Brain Project
+ *                          Stefan.Eilemann@epfl.ch
+ *
+ * This file is part of Brion <https://github.com/BlueBrain/Brion>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#pragma once
+
+#include "../constants.h"        // used inline
+#include "../morphologyPlugin.h" // base class
+
+#include <servus/serializable.h> // base class
+
+#include <future>
+
+namespace brion
+{
+namespace plugin
+{
+class MorphologyZeroEQ : public MorphologyPlugin, public servus::Serializable
```

Comment:
Let me see what I can do. It's both a serializable morphology (for brain & ZeroEQ) and the ZeroEQ plugin.